Src/BootCamp.Chapter/TextBoxBuilder.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public class TextBoxBuilder
+    {
+        private readonly string _message;
+        private readonly TextTableAttribute _textTableAttributes;
+        private readonly int _messageLength;
+        private readonly int _numberOfLines;
+        private readonly int _numberOfMainLine;
+        private readonly string _paddingString;
+        private StringBuilder sb = new StringBuilder();
+        private const int DivideByTwo = 2;
```

Comment:
DivideByTwo or Addone does not make anything better. You need to describe those constants in a logical why (what are they used for?). 

You did good with `NumberOfLinesWithoutPadding`